lib/topological_inventory/openshift/parser.rb

```diff
@@ -26,15 +27,8 @@ class Parser
     attr_accessor :collections, :resource_timestamp, :openshift_host, :openshift_port
 
     def initialize(openshift_host:, openshift_port: 8443 )
-      entity_types = [:containers, :container_groups, :container_nodes, :container_projects, :container_images,
```

Comment:
I wonder if this belongs to common. We should probably list here only collections that are used? @agrare ?

Reply:
@Ladas entity types are not specified in common gem. You can still use `add_collection`, but if we have TopologicalInventoryIngressApiClient's models defined by swagger, then this only defines what was previously defined in api gem. 
So storage checks, if collection with the same model name exists and creates it, otherwise raises NameError

Reply:
explicit call to `add_collection` is meant only when we will need to define some non-standard collections

Reply:
one example:
```
@collections[:vms].build(:name => "xxx") # it creates Inventory Collection and data at the same time
@collections[:nonsense].build(:name => "xxx") # it throws NameError
```

Reply:
Hm I'm okay with this being defined on demand as long as it actually checks against the actual list of collections and you can't just define garbage

Reply:
now it's working this way, any access to `collections` without corresponding`TopologicalInventoryIngressApiClient's` model raises `NameError`